feature-libs/organization/administration/components/shared/message/base-message.component.ts

```diff
@@ -40,6 +40,8 @@ export abstract class BaseMessageComponent implements OnInit {
 
   close(): void {
     this.messageData.events.next({ close: true });
+    delete this.messageData.message;
```

Comment:
how about adding new `delete` method to `message.service.ts` and using it here?

Reply:
I cant do that there, seems as lack of access to this property